### ISSUE TITLE
RIT FOSS Student Ticket Index

### ISSUE THREAD

**Issue by isabela-pf** [jupyter/accessibility#77]
🏁 Welcome to jupyter/accessibility, and thank you so much for your interest! The following is a list of issues across the Jupyter ecosystem that might be good entry points into contributing to Jupyter accessibility needs. 

## Accessibility fixes

- [JupyterLab - Modal dialogs are not accessible #6579](https://github.com/jupyterlab/jupyterlab/issues/6579) (has prior unmerged work for reference)
- [JupyterLab -  Alt Text and Marking Elements Decorative #9682](https://github.com/jupyterlab/jupyterlab/issues/9682)
- [JupyterLab -  Place region of elements in a logical reading order #9696 ](https://github.com/jupyterlab/jupyterlab/issues/9696) (has a suggested fix)
- [jupyter.org -  Is there a better alternative to Google Calendar for events? #574 ](https://github.com/jupyter/jupyter.github.io/issues/574) (accessibility is not listed on the issue, but the Google Calendar plugin is not accessible and future implementations need to be)

## Evaluating projects
- [RetroLab -  Run an accessibility audit on this UI #80](https://github.com/jupyterlab/retrolab/issues/80)
- Review [the jupyter/accessibility site](https://jupyter-accessibility.readthedocs.io/en/restructure/README.html) for accessibility. This is a popular documentation theme and reviewing its accessibility will help us know what improvements we need to make to help many projects.

## Proposing templates and/or Documentation

- [jupyter/accessibility -  Contribution guide #47](https://github.com/jupyter/accessibility/issues/47)
- [jupyter/accessibility - Make a github issue template for audits #22 ](https://github.com/jupyter/accessibility/issues/22)
- [jupyter/accessibility -  Accessibility docs best practices #15](https://github.com/jupyter/accessibility/issues/15)

⭐ If you are interested in working on an issue, it's best to let the community know so we can support each other. Commenting your interest on the original issue (best practice) and on this issue (optional) is a good way to communicate this.

Please don't hesitate to reach out for help and guidance!

**Comment by isabela-pf** [comment]
We can close it and reference it if needed.
